wafer/templates/wafer/base_form.html

```diff
@@ -1,12 +1,15 @@
 {% extends "wafer/base.html" %}
 {% load i18n %}
 {% load crispy_forms_tags %}
+{% load static from staticfiles %}
 {% block extra_head %}
   {{ form.media.css }}
 {% endblock %}
 {% block content %}
   {% crispy form %}
 {% endblock %}
 {% block extra_foot %}
+  <script type="text/javascript" src="{% static 'admin/js/jquery.init.js' %}">
```

Comment:
What does this change do?

Reply:
it makes any select2 fields work at all. for some reason, jquery was not being loaded in any non-admin pages.

Reply:
There is this in the base template though -- https://github.com/CTPUG/wafer/blob/master/wafer/templates/wafer/base.html#L37 ?

There is also this weird thing inside the schedule editor -- https://github.com/CTPUG/wafer/blob/master/wafer/static/js/scheduledatetime.js ?

Maybe we need a snippet like that inside talk-categorization.js somehow?

Reply:
the issue is that easy_select2 requires `django` and `django.jQuery` to be defined; see  `easy_select2/static/easy_select2/js/easy_select2.js `.

`django` and `django.jQuery` are defined in `admin/js/jquery.init.js`. It seems like easy_select2 (or at least, its latest version) is only meant for the admin, and we are kind of abusing it in arbitrary forms. or maybe that's just an oversight in easy_select2

Reply:
to clarify: this fixes the __existing__ select2 fields in wafer, not the ones I am adding in this PR (as for those I am calling select2 directly due to the reasons explained above)

Reply:
Could you add a comment to the template describing why this is here? After that I think I'm happy with the PR.

Reply:
Also, thanks for sorting this out!